src/main/kotlin/org/jqassistant/tooling/intellij/plugin/editor/config/toolwindow/LoadingPanel.kt

```diff
@@ -0,0 +1,15 @@
+package org.jqassistant.tooling.intellij.plugin.editor.config.toolwindow
+
+import java.awt.BorderLayout
+import javax.swing.ImageIcon
+import javax.swing.JLabel
+import javax.swing.JPanel
+
+class LoadingPanel : JPanel() {
+    init {
+        layout = BorderLayout()
+        val loadingIcon = ImageIcon(javaClass.getResource("/icons/loading.gif"))
```

Comment:
```suggestion
        val loadingIcon = AnimatedIcon.Default()
```
Using the AnimatedIcon class works better on HighDPI screens and is commonly used in the intellij codebase